src/jit/instrsxarch.h

```diff
@@ -491,11 +489,14 @@ INST3(vpsrlvq,          "psrlvq",           IUM_WR, BAD_CODE,     BAD_CODE,
 INST3(vpsravd,          "psravd",           IUM_WR, BAD_CODE,     BAD_CODE,     SSE38(0x46),                             INS_Flags_IsDstDstSrcAVXInstruction)    // Variable Bit Shift Right Arithmetic
 INST3(vpsllvd,          "psllvd",           IUM_WR, BAD_CODE,     BAD_CODE,     SSE38(0x47),                             INS_Flags_IsDstDstSrcAVXInstruction)    // Variable Bit Shift Left Logical
 INST3(vpsllvq,          "psllvq",           IUM_WR, BAD_CODE,     BAD_CODE,     SSE38(0x47),                             INS_Flags_IsDstDstSrcAVXInstruction)    // Variable Bit Shift Left Logical
+INST3(vperm2i128,       "perm2i128",        IUM_WR, BAD_CODE,     BAD_CODE,     SSE3A(0x46),                             INS_Flags_IsDstDstSrcAVXInstruction)    // Permute 128-bit halves of input register
```

Comment:
Why did the existing ones get moved around in file? (mostly wanting to know if it just happened or if it was for an explicit reason)

Reply:
I'd agree that, unless there's a reason, it would be better to leave them where they were.

Reply:
The only reason was to keep all Vex permute instructions together, if that is not best reason I will correct.

Reply:
> in order to minimize merge conflicts on the various flag refactorings that are being investigated.

OK I see the problem, reverting the change.